### PR TITLE
Dev inflib

### PR DESCRIPTION
**Pull request by InfinityRaider**
Port to InfinityLib

### CODE REVIEW

**Comment by TehNut** [comment]
Not sure where else to put this, but you're missing `@Retention(RetentionPolicy.RUNTIME)` in [`InfinityMod`](https://github.com/InfinityRaider/InfinityLib/blob/master/src/main/java/com/infinityraider/infinitylib/InfinityMod.java).

**Comment by InfinityRaider** [comment]
Thanks, also need to put a dependency in agricraft's Mod annotation

File: src/main/java/com/infinityraider/agricraft/renderers/blocks/RenderChannel.java [hunk 1]
```diff
@@ -48,24 +48,49 @@ protected void renderBottom(ITessellator tessellator, TextureAtlasSprite matIcon
...
-			// draw an edge
-			tessellator.drawScaledPrism(5, 5, 4, 11, 12, 5, matIcon);
-		} else {
-			// extend bottom plane and side edges
-			tessellator.drawScaledPrism(4, 4, 0, 12, 5, 4, matIcon);
-			tessellator.drawScaledPrism(4, 5, 0, 5, 12, 5, matIcon);
-			tessellator.drawScaledPrism(11, 5, 0, 12, 12, 5, matIcon);
-		}
+	protected void renderSide(ITessellator tessellator, T channel, EnumFacing dir, boolean connect, TextureAtlasSprite matIcon) {
+        switch(dir) {
```
Comment (TehNut) [inline src/main/java/com/infinityraider/agricraft/renderers/blocks/RenderChannel.java]: Looks like formatting went wrong here.

Comment (InfinityRaider) [inline src/main/java/com/infinityraider/agricraft/renderers/blocks/RenderChannel.java]: Did it? I 'm not noticing anything off.

Comment (TehNut) [inline src/main/java/com/infinityraider/agricraft/renderers/blocks/RenderChannel.java]: A bit late, but on mobile it looked like this line was indented with only a single space. *shrug*

Comment (InfinityRaider) [inline src/main/java/com/infinityraider/agricraft/renderers/blocks/RenderChannel.java]: No wait, you're right, the switch is indeed indented wrongly.

**Comment by RlonRyan** [comment]
Wow that's a lot of failed builds in a single PR... Lol...

**Pull request merged**